src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -484,6 +484,11 @@ Copyright (c) .NET Foundation. All rights reserved.
                                     Value="$(ThreadPoolMaxThreads)" />
   </ItemGroup>
 
+  <PropertyGroup>
```

Comment:
This feels like an odd spot for this property group. Would it make sense to go higher in the file, before any `<Targets>`? Like maybe around line 100?

Reply:
Thinking a bit more, should this switch get set for `PublishTrimmed=true` instead of `non-Debug` configurations? Does it have any effect on non-trimmed applications?

Reply:
@pranavkm explained to me that setting this for PublishTrimmed=true would be a bad idea because then it would be disabled even during `dotnet run` and when debugging. So I realize now that was not a good idea.